drivers/watchdog/wdt_nrfx.c

```diff
@@ -124,6 +120,11 @@ static int wdt_nrf_install_timeout(struct device *dev,
 	}
 
 	if (m_allocated_channels == 0) {
+
+		if (!(cfg->window.max > WDT_NRF_MIN_TIME_MS)) {
```

Comment:
`if (cfg->window.max < WDT_NRF_MIN_TIME_MS)`
Or would that look too trivial?

Reply:
I will change it to check if value is greater than zero and add proper comment